### ISSUE TITLE
Cloth3D dataset

### ISSUE THREAD

**Issue by runnerfavourite** [saic-vul/point_based_clothing#8]
Hello sir, I am a student who is studying Point Cloud. Recently discovered papers 《Cloth3d: Clothed 3d humans》Dataset download address resource can not be downloaded successfully, can you provide other download channels for this dataset?

**Comment by taruntiwar** [comment]
Yes, I also requested you guys for model checkpoints, Can you please give access to the download checkpoint?
Thanks

**Comment by izakharkin** [comment]
> Hello sir, I am a student who is studying Point Cloud. Recently discovered papers 《Cloth3d: Clothed 3d humans》Dataset download address resource can not be downloaded successfully, can you provide other download channels for this dataset?

Hello @runnerfavourite , we downloaded this dataset a while ago and did not try it since then. If there are some problems with getting it I advise you to write directly to the competition organizers https://chalearnlap.cvc.uab.cat/main-organizers and/or to the authors of the corresponding dataset paper: https://arxiv.org/abs/1912.02792

**Comment by izakharkin** [comment]
Hello @taruntiwar , the absence of access is a known issue, it will be fixed shortly - I will reply in this issue. Thank you

**Comment by izakharkin** [comment]
Hi @taruntiwar, `checkpoints/` and `samples/` download links in [README.md](https://github.com/saic-vul/point_based_clothing/blob/main/README.md#download-data) were fixed, feel free to ask any other questions.

**Comment by izakharkin** [comment]
Hi @runnerfavourite , it seems that the [website](https://chalearnlap.cvc.uab.cat/dataset/38/description/) is now alive. You can download the dataset by following the links at the bottom of the page.
